src/components/userprofile/UserProfileInfo/UserProfileInfo.module.scss

```diff
@@ -0,0 +1,62 @@
+.infoSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 2rem;
+  padding: 0 2rem;
+  text-align: center;
+
+  @media (min-width: $screen-lg-min) {
+    flex: 0 0 300px;
+    align-items: flex-start;
+    min-width: 300px;
+    max-width: 300px;
```

Comment:
I smell hacks. :D I would normally not accept this. But we don’t have the luxury of time.